src/main/resources/io/jenkins/plugins/designlibrary/Localization/index.properties

```diff
@@ -0,0 +1,13 @@
+titlejava=Localize text in Java to show it in the user''s language
+howtojava=In order to make texts defined in Java localizable you have to create a file <em>Messages.properties</em> in the resources folder of your Java package. \
```

Comment:
this is a lot of content it might be better to split it up with just whats needed for examples?